ethcore/sync/src/api.rs

```diff
@@ -647,6 +647,8 @@ pub struct NetworkConfiguration {
 	pub ip_filter: IpFilter,
 	/// Client version string
 	pub client_version: String,
+	/// HTTPS Proxy
+	pub https_proxy: Option<String>,
```

Comment:
I am not quite sure why there are two (here and https://github.com/paritytech/parity-ethereum/pull/9303/files#diff-034858a645d5e2357f9460db644109d3R207) NetworkConfiguration structs in the project, but I've added the fields to both.

Also, I am not sure if it really should be Option.

Reply:
I think it makes sense because it is/should be optional to use a ```https proxy```